pipeline/run-tests-in-docker-linux.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This template:
+# - builds and runs a given target in DOCKERFILE
+# - copies /app/test-results & /app/drop back to the base agent
+# - requires you to publish test results or artifacts in the consuming pipeline
+# - only runs on linux agents
```

Comment:
My initial reading of this before looking at the source was "this template is only applicable to linux agents, so that's the only place you should invoke it from", rather than "this template may be applied on any agent OS but will no-op on non-linux"; maybe clarify the comment?